File: Study Note/Algorithm Problems/SWEA/D3/5549 홀수일까 짝수일까 (pass)/SWEA_5549.py
import sys
sys.stdin = open("input.txt", "r")

T = int(input())
for test_case in range(1, T + 1):
    # ///////////////////////////////////////////////////////////////////////////////////
    '''

        이 부분에 여러분의 알고리즘 구현이 들어갑니다.

    '''
    # ///////////////////////////////////////////////////////////////////////////////////
    N = input()
    if int(N[-1]) % 2:
        print('#{} Odd'.format(test_case))
    else:
        print('#{} Even'.format(test_case))

File: Study Note/Algorithm Problems/SWEA/D3/5549 홀수일까 짝수일까 (pass)/SWEA_5549.c
#include <stdio.h>
int main(void)
{
	int test_case;
	int T, i, n;
    char *str;
	
	scanf("%d", &T);
	
	for (test_case = 1; test_case <= T; ++test_case)
	{
		scanf("%s", str);
		i = 0;
        while (str[i] != '\0')
            i++;
        n = str[i-1] - '0';
        if (n % 2) printf("#%d Odd\n", test_case);
        else printf("#%d Even\n", test_case);
	}
	return 0;
}